app/controllers/miq_ae_class_controller.rb

```diff
@@ -1749,7 +1753,7 @@ def domains_priority_edit
       domains = @edit[:new][:domain_order].reverse!.collect do |domain|
         MiqAeDomain.find_by_name(domain.split(' (Locked)').first).id
       end
-      MiqAeDomain.reset_priority_by_ordered_ids(domains)
+      MiqAeDomain.reset_priority_by_ordered_ids(domains, current_tenant)
```

Comment:
@gmcculloug @dclarizio 
We don't need the tenant anymore on this call since we already have a list of domain ids. I have removed the need for tenant.